src/Table.luna

```diff
@@ -769,18 +1122,87 @@ class Table:
         quart3 = col.quantile 0.75
         Table.fromColumns [mean, std, min, quart1, median, quart3, max]
 
+    # Returns table with unique values counts for selected column.
```

Comment:
Not sure if "value count" is really clear. Perhaps "occurrence count" or sth like this would be better.